celer/PN_logreg.pyx

```diff
@@ -35,6 +35,9 @@ def newton_celer(
     else:
         dtype = np.float32
 
```

Comment:
suggestion: add a comment above saying "Elastic net regularization currently not supported for PN logreg"